compiler/pom.xml

```diff
@@ -70,6 +80,7 @@
               <exclude>com.google.auto.service:auto-service</exclude>
               <exclude>com.google.auto.value:auto-value</exclude>
               <exclude>com.google.dagger:dagger</exclude>
+              <exclude>com.google.guava</exclude>
```

Comment:
This apparently breaks the Java 8 build; probably because the maven-shade-plugin rewrites references to `com.google.common` classes into `bullet.thirdparty.com.google.common`, but doesn't relocate those classes as we're telling it to exclude them here.
I suppose a better `<relocation>` configuration could fix it, but better leave that for another PR as we were already shading Guava classes anyway, so this is a different change than the one we're currently interested in.